install/kubernetes/cilium/values.yaml.tmpl

```
@@ -1048,28 +1048,67 @@ hubble:
   # -- Unix domain socket path to listen to when Hubble is enabled.
   socketPath: /var/run/cilium/hubble.sock
 
-  # -- Configures the redact options for Hubble.
-  # Example:
-  #
-  #   redact:
-  #     enabled: true
-  #     http:
-  #       urlQuery: true
-  #     kafka:
-  #       apiKey: false
-  #
-  # You can specify the options from the helm CLI:
-  #
-  #   --set hubble.redact.enabled="true"
-  #   --set hubble.redact.http.urlQuery="true"
-  #   --set hubble.redact.kafka.apiKey="false"
-  #
+  # -- Enables redacting sensitive information present in Layer 7 flows.
   redact:
     enabled: false
     http:
-        urlQuery: false
+      # -- Enables redacting URL query (GET) parameters.
+      # Example:
+      #
+      #   redact:
+      #     enabled: true
+      #     http:
+      #       urlQuery: true
+      #
+      # You can specify the options from the helm CLI:
+      #
+      #   --set hubble.redact.enabled="true"
+      #   --set hubble.redact.http.urlQuery="true"
+      urlQuery: false
+      headers:
+        # -- List of HTTP headers to allow -- headers not matching will be redacted. Note: `allow` and `deny` lists cannot be used both at the same time, only one can be present.
```

Having another `--` in the docstring breaks generation of the README and documentation, hence why they were not present in the new version of the README after update, so we need to remove that:
```suggestion
        # -- List of HTTP headers to allow: headers not matching will be redacted. Note: `allow` and `deny` lists cannot be used both at the same time, only one can be present.
```